Description: Drop sphinxcontrib.apidoc extension until package is available.
Author: Corey Bryant <corey.bryant@canonical.com>
Forwarded: non-needed

--- a/doc/source/conf.py
+++ b/doc/source/conf.py
@@ -50,7 +50,7 @@ extensions = ['stevedore.sphinxext',
               'oslo_config.sphinxext',
               'oslo_config.sphinxconfiggen',
               'openstackdocstheme',
-              'sphinxcontrib.apidoc',
+              #'sphinxcontrib.apidoc',
               ]
 
 # openstackdocstheme options
